backend/entities/Models/ProjectFilter.cs

```diff
@@ -39,10 +39,10 @@ public class ProjectFilter : PageFilter
         public bool? CreatedByMe { get; set; }
 
         /// <summary>
-        /// get/set - Filter by workflow - ACCESS-DISPOSAL.
+        /// get/set - Filter by workflow - ACCESS-DISPOSAL or ACCESS-EXEMPTION.
         /// </summary>
         /// <value></value>
-        public bool? AccessDisposal { get; set; }
+        public bool? AccessWorkflow { get; set; }
```

Comment:
I'm just realizing we're using the work `ACCESS` instead of `ASSESS`...  Ugh

Reply:
haha i thought it was supposed to be that but then I thought I was missing something!